controls/V-72047.rb

```diff
@@ -61,21 +67,11 @@
 
 # chgrp root <directory>"
 
-  # @todo - add option for app group associated with dir?
   ww_dirs = command('find / -xdev -perm -002 -type d -exec ls -ld {} \;').stdout.split("\n")
```

Comment:
This seems a little aggressive, I'll have to dig around for my code that is more comprehensive. It's in one of the checks already in place.

Reply:
@trevor-vaughan did you every find your code that should be used here?

Reply:
I think this is actually all set. The current version of this control checks for world-writable directories that are not owned by root, sys, bin, or application groups.